scripts/utils/bin/fetch_rollup_config.rs

```diff
@@ -139,7 +145,11 @@ async fn update_l2oo_config() -> Result<()> {
     let (_, agg_vkey) = prover.setup(AGGREGATION_ELF);
     let aggregation_vkey = agg_vkey.vk.bytes32();
 
+    #[cfg(feature = "celestia")]
+    let (_, range_vkey) = prover.setup(CELESTIA_RANGE_ELF_EMBEDDED);
+    #[cfg(not(feature = "celestia"))]
     let (_, range_vkey) = prover.setup(RANGE_ELF_EMBEDDED);
+
```

Comment:
Rather than features, in the config scripts it's probably better to do a match statement with flags (as it won't require re-compiling)